lib/process-services/src/lib/form/widgets/people/people.widget.spec.ts

```diff
@@ -64,19 +64,20 @@ describe('PeopleWidgetComponent', () => {
         it('should return full name for a given model', () => {
             const model = {
                 firstName: 'John',
-                lastName: 'Doe'
+                lastName: 'Doe',
+                email: 'john@doe.com'
             };
-            expect(widget.getDisplayName(model)).toBe('John Doe');
+            expect(widget.getDisplayName(model)).toBe('John Doe <john@doe.com>');
         });
 
         it('should skip first name for display name', () => {
```

Comment:
missing the case when email is not provided (I guess admins don't have that, i.e. superadmin)

Reply:
not relevant anymore